packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -4,28 +4,18 @@ import { Percent } from '@cardano-sdk/util';
 import { PoolIdHex } from './primitives';
 import { PoolParameters } from './PoolParameters';
 
-/**
- * Stake quantities for a Stake Pool.
- */
+/** Stake quantities for a Stake Pool. */
```

Comment:
There are a lot of formatting changes for doc comments. I'm fine with new format, but it would be best to do it in a separate commit:
- enforce it via our tooling (probably eslint)
- apply the new format to entire codebase

Reply:
Once merged #959 , simply rebasing this branch should solve this as well

Reply:
9bb6579a4 should solve this